app/models/work.rb

```diff
@@ -0,0 +1,35 @@
+class Work < ApplicationRecord
+  has_many :upvotes
+
+  validates :title, presence: true, uniqueness: true, length: { minimum: 1}
+
+  validates :creator, presence: true
+
+  validates :publication_year, presence: true, numericality: {only_integer: true, greater_than: 0, less_than_or_equal_to: Date.current.year}
+
+  KINDS = ["Album", "Book", "Movie"]
```

Comment:
I like that you name the categories in this array as a constant!

You could push this further to make it an array of symbols